tests/setup/before.ts

```diff
@@ -0,0 +1,7 @@
+import * as chai from 'chai';
+import enableStrictlyEqual from '@silvermine/chai-strictly-equal';
+
+let jsdom: any = require('jsdom-global');
```

Comment:
Just checking, do we need jsdom? I think this is from the auto-generated template code